tests/Functional/Execution/ExecutionTest.php

```diff
@@ -84,17 +85,20 @@ public function testAcceptAnObjectWithNamedPropertiesAsArguments()
                 ])
         ]);
 
-        $rootValue    = 'rootValue';
-        $documentNode = parse('query Example { a }');
+        $rootValue = 'rootValue';
+        $document  = 'query Example { a }';
```

Comment:
Ain't this `$source`?

Reply:
It's DocumentNode, so that I call it document : D, but it's can be source.